src/dashboard/components/Login/LoginBox.tsx

```diff
@@ -14,9 +15,19 @@ import Emoji from "../Emoji";
 import styles from "./LoginBox.module.scss";
 // public
 import Logo from "/public/logo/black/ecoseller.io.svg";
+// axios
+import { axiosPrivate } from "../../src/common/axiosPrivate";
```

Comment:
I don't want to dig into it too much and do nitpicking but maybe we could put axiosInstance into `/utils` folder so that we could import it like `@/utils/axiosPrivate`?

Reply:
Agreed.